source/blog/2017-05-12-oo-standalone-registry.md

```diff
@@ -0,0 +1,148 @@
+
+It turns out that deploying the stand-alone registry on a single Fedora 25 Atomic Host system is quite straight-forward and can quickly provide a usable registry.
+
+READMORE
+
+A good portion of this guide was lifted and adapted from [Dusty Mabe's](https://twitter.com/dustymabe) excellent guide on [installing OpenShift Origin on Fedora 25 Atomic Host](http://www.projectatomic.io/blog/2016/12/part1-install-origin-on-f25-atomic-host/).  He also assisted me along the way with some questions, so most of the credit goes to him for figuring out the hard stuff!
+
+Environment
+============
+For the purposes of this guide, we are going to deploy a stand-alone registry using the 'all-in-one' topology, where we host all the components on a single system.  The OpenShift Origin docs have some [hardware requirements](https://docs.openshift.org/latest/install_config/install/stand_alone_registry.html#registry-minimum-hardware-requirements) for this kind of install; they seem rather generous, so you might be able to get away with a less powerful system.  As with most things, your mileage may vary.
+
+In addition to your dedicated system for the registry, you'll also need to have Ansible 2.2 installed on your workstation (or wherever you choose to run the Ansible-based installer).
+
+Once your Fedora 25 Atomic Host system has been provisioned, make sure it is upgraded to the latest version by using the `atomic host upgrade` command.
+
+Additionally, I decided I would configure Docker to use the `overlay2` storage driver for my install.  This is completely optional, but if you decide to make this change you can use the [instructions in a previous blog post](http://www.projectatomic.io/blog/2017/05/migrate-fedora-atomic-host-to-overlay2/) to do so.
+
+Prepping the Installer
+=======================
+OpenShift Origin provides an excellent [Ansible-based](https://www.ansible.com/) installer that we will used to deploy the registry.  So the first thing to do is checkout the installer from the [GitHub repo](https://github.com/openshift/openshift-ansible).  At the time of this writing, I used a specific version ([3.4.24-1](https://github.com/openshift/openshift-ansible/releases/tag/openshift-ansible-3.4.24-1)) of the repo to ensure it would work correctly for me.  We'll checkout the repo to that version as part of this process.
+
+    $ git clone https://github.com/openshift/openshift-ansible.git
+    $ cd openshift-ansible
+    $ git checkout openshift-ansible-3.4.24-1
+    HEAD is now at 90f6a70... Automatic commit of package [openshift-ansible] release [3.6.24-1].
+
+The next step is to prepare the inventory file for the installer.  My dedicated system for the registry has a 'public' IP address of `10.8.172.199`, so we use that value throughout.  (I'm using a VM in a private OpenStack instance, so the 'public' IP address is only visible on our corporate network and is not reachable from the Internet.)
```

Comment:
Could you mention the use of [xip.io](http://xip.io/) here? It's an essential bit of magic to make dns routing work. How about...

"Notice I'm using the [xip.io](http://xip.io/) service to resolve wildcard DNS queries."